config/Dockerfiles/cloud-image-compose/virt-customize.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/sh
+
+set -xeuo pipefail
+
+# A shell script that pulls the latest nightly from rcm
+# and uses virt-customize to inject rpms into it. It 
+# outputs a new qcow2 image for you to use.
+#
+# Note: This is only configured for rhel8 currently
```

Comment:
Remove this line, I'm not sure what is supposed to be secret and would prefer to be safe